src/main/java/core/basesyntax/dao/FruitStorageDao.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.dao;
+
+import java.util.HashMap;
+
+public interface FruitStorageDao {
+    void addFruit(String key, Integer value);
+
+    void modifyValue(String key, Integer value);
+
+    Integer get(String key);
+
+    HashMap<String, Integer> getStorage();
```

Comment:
We should avoid using large interfaces and break those in small ones (Interface segregation, but I am not 100% sure)

Reply:
In this case as I understood from the video all methods working with database and have like common task - work with database and provide tools for this purpose to services. So probably I'll ask this at the next QnA